CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [2.2.2] - 2017-09-28
```

Comment:
To say the truth for me it feels like release of a new feature, so it could probably be 2.3.0
Or is it already published as 2.2.2 ?

Reply:
I can rename it in release notes and in change log, but the tag name still will be 2.2.2, okay?

Reply:
I can even create a new tag )

Reply:
No, I'm totally fine with 2.2.2
I think there is now need to redo this.